website/docs/deployment.md

```diff
@@ -9,7 +9,7 @@ To build the static files of your website for production, run:
 npm run build
 ```
 
-Once it finishes, you should see the production build under the `build/` directory.
+Once it finishes, you might want to see the production build under the `build/` directory.
```

Comment:
This is not about telling the user what to do. This is more of telling the user what to expect. I think we can change it to "you will" instead.